### PR TITLE
Align occ status with status.php

### PR DESCRIPTION
**Pull request by juliusknorr**
Make sure that all fields that are available in status.php are also exposed when running `occ status`. Especially `needsDbUpgrade` might be useful for automated checks.

### CODE REVIEW

**Comment by juliushaertl** [comment]
Failures unrelated

File: core/Command/Status.php [hunk 1]
```diff
@@ -38,11 +40,17 @@ protected function configure() {
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$config = \OC::$server->getConfig();
+		$defaults = \OC::$server->getThemingDefaults();
 		$values = [
-			'installed' => (bool) \OC::$server->getConfig()->getSystemValue('installed', false),
-			'version' => implode('.', \OCP\Util::getVersion()),
-			'versionstring' => \OC_Util::getVersionString(),
+			'installed' => (bool) $config->getSystemValue('installed', false),
```
Comment (blizzz) [inline core/Command/Status.php:46]: maybe even `getSystemValueBool`?

File: core/Command/Status.php [hunk 2]
```diff
@@ -38,11 +40,17 @@ protected function configure() {
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$config = \OC::$server->getConfig();
+		$defaults = \OC::$server->getThemingDefaults();
```
Comment (blizzz) [inline core/Command/Status.php:44]: DI?

**Review by blizzz — APPROVED** [review]

**Review by CarlSchwan — APPROVED** [review]
Code looks good to me also tested

**Review by artonge — APPROVED** [review]

**Pull request merged**